plugins/rapid7_insightvm_cloud/unit_test/test_vuln_search.py

```diff
@@ -10,8 +10,8 @@
 from icon_rapid7_insightvm_cloud.actions.vuln_search.schema import Input
 from icon_rapid7_insightvm_cloud.connection.schema import Input as ConnectionInput
 from unittest.mock import patch
-from unit_test.utils import Utils
-from unit_test.mock import (
+from utils import Utils
```

Comment:
to validate the schemas we should add the tests Igor suggested that will validate against a mocked output? (pinned in soarcerers-scrum channel).